app/controllers/users/destroy.js

```diff
@@ -1,5 +1,6 @@
 import DestroyController from 'amber-ui/controllers/application/destroy';
 
-export default DestroyController.extend({
-  successTransitionTarget: 'users.index',
-});
+export default class UserDestroyController extends DestroyController {
```

Comment:
When clicking on the delete button it shows the following error message above the button `Er ging iets fout bij het verwijderen: Error: Ember Data Request DELETE /api/v1/users/38 returned a 404 Payload (application/vnd.api+json; charset=UTF-8) [object Object]` en de gebruiker wordt niet verwijderd

Reply:
this is not introduced by this PR. in fact, it seems to be the case because the user model doesn't have a delete endpoint in amber-api.
I get the following exception in the network tab:
`#<ActionController::RoutingError: No route matches [DELETE] "/v1/users/50">`
![image](https://user-images.githubusercontent.com/32799956/198140306-6ab7c03e-61d2-44ca-9334-4c3f37d37bf0.png)

Reply:
#642